playwright/README.md

```diff
@@ -0,0 +1,106 @@
+# Playwright Test Component
+
+## How to write a test
+
+1. You need to select the component for which you want to write tests
+2. In the component folder, create a folder `__tests__`
+3. Create a file in it `NameComponent.visual.test.tsx`
+4. In the file to make imports:
+
+   ```ts
+   import React from 'react';
+
+   import {expect, test} from '@playwright/experimental-ct-react';
+
+   import {MyTestedComponent} from '../MyTestedComponent';
+   ```
+
+5. Writing a test:
```

Comment:
4 and 5 steps can be merged into one

Reply:
I've meant get code from 4 and put it into 5. Title `In the file to make imports` should be removed also